cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -1108,6 +1108,39 @@ pub(crate) fn define(
         .operands_out(vec![a]),
     );
 
+    let mask = &operand_doc(
+        "mask",
+        &imm.uimm128,
+        "The 16 immediate bytes used for selecting the elements to shuffle",
+    );
+    let Tx16 = &TypeVar::new(
+        "TxN",
+        "A SIMD vector type",
```

Comment:
... and maybe tweak this comment too (noting it's x16 right now, but could be changed later for other SIMD types).